examples/README.md

```diff
@@ -27,6 +27,8 @@ This directory contains a collection of examples that demonstrate the use of the
     implementation.
   + `tower-load`: Demonstrates how dynamically reloadable filters can be used to
     debug a server under load in production.
+  + `journald`: Demonstrates how to use `fmt` and `journald` layers to output to
+    both the terminal and the system journal.
```

Comment:
Should this maybe note that the example won't work on systems without journald? Maybe it's obvious enough that we don't need to...

Reply:
I'll tweak it to only use journald if available.

Reply:
Great, that's probably something that's good to show an example of anyway.